index.html

```diff
@@ -2852,7 +2852,7 @@ <h2>Definition of Roles</h2>
                 <rref>button</rref> from its descendant content.
               </li>
             </ul>
-            <pre class="example highlight">
+            <pre class="example highlight html">
           &lt;label id="tag_label" for="tag_combo">Tag&lt;/label>
```

Comment:
Indentation is off on this one I think.
```suggestion
      &lt;label id="tag_label" for="tag_combo">Tag&lt;/label>
```

Reply:
I suspect prettier will just undo this.